packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -1524,7 +1524,7 @@ class ClipboardStatusNotifier extends ValueNotifier<ClipboardStatus> with Widget
       final ClipboardStatus clipboardStatus = data != null && data.text != null && data.text.isNotEmpty
           ? ClipboardStatus.pasteable
           : ClipboardStatus.notPasteable;
-      if (clipboardStatus == value) {
+      if (clipboardStatus == value || _disposed) {
```

Comment:
Let's move the if (_disposed) check to be the first thing in this callback since we don't have to do any of the other work if that happens.

Reply:
Smart, will do.